build/devices/esp32/targets/m5atom_s3/manifest.json

```diff
@@ -2,7 +2,7 @@
         "build":{
                 "ESP32_SUBCLASS": "esp32s3",
                 "USE_USB": "1",
-                "SDKCONFIGPATH": "$(MODDABLE)/build/devices/esp32/targets/atoms3/sdkconfig"
+                "SDKCONFIGPATH": "$(MODDABLE)/build/devices/esp32/targets/m5atom_s3/sdkconfig"
```

Comment:
A recent enhancement to mcconfig can simplify this and eliminate the dependency on the directory name. Please try using this instead:

```
                "SDKCONFIGPATH": "./sdkconfig"
```